recipes/compas_libigl/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "compas_libigl" %}
+{% set version = "0.2.2" %}
+{% set pybind11_version = "2.10.4" %}
+{% set libigl_version = "2.2.0" %}
+{% set hash = "e3ee9a9915cb05559ec4c71885ae91b635b28b4ee75753a43bc1eddadc9624da" %}
+{% set pybind11_hash = "832e2f309c57da9c1e6d4542dedd34b24e4192ecb4d62f6f4866a737454c9970" %}
+{% set libigl_hash = "b336e548d718536956e2d6958a0624bc76d50be99039454072ecdc5cf1b2ede5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/compas-dev/compas_libigl/archive/v{{ version }}.tar.gz
+    fn: compas_libigl-{{ version }}.tar.gz
+    sha256: {{ hash }}
+  - url: https://github.com/pybind/pybind11/archive/v{{ pybind11_version }}.tar.gz
```

Comment:
It would be preferable to use the pybind11 from codna-forge. Can you file an issue upstream please?